server_addon/maya/client/ayon_maya/plugins/publish/collect_yeti_rig.py

```diff
@@ -271,22 +272,23 @@ def get_sequence(self, filepath, pattern="%04d"):
             pattern (str): The pattern to swap with the variable frame number.
 
         Returns:
-            list: file sequence.
+            list[str]: file sequence.
 
         """
-        import clique
-
-        escaped = re.escape(filepath)
-        re_pattern = escaped.replace(pattern, "-?[0-9]+")
-
+        filename = os.path.basename(filepath)
+        re_pattern = re.escape(filename)
+        re_pattern = re_pattern.replace(re.escape(pattern), "-?[0-9]+")
         source_dir = os.path.dirname(filepath)
-        files = [f for f in os.listdir(source_dir)
-                 if re.match(re_pattern, f)]
-
-        pattern = [clique.PATTERNS["frames"]]
-        collection, remainder = clique.assemble(files, patterns=pattern)
-
-        return collection
+        files = [f for f in os.listdir(source_dir) if re.match(re_pattern, f)]
+        collections, _remainder = clique.assemble(
+            files,
+            patterns=[clique.PATTERNS["frames"]],
+            minimum_items=1)
+            
+        if len(collections) > 1:
+            raise ValueError(f"Multiple collections found for {files}. This is a bug.")
```

Comment:
I wonder if - this would be more easier to debug in the logs if instead of the files it would print the `collections` so you could see which collections may have been generated.

```suggestion
        if len(collections) > 1:
            raise ValueError(f"Multiple collections found for {collections}. This is a bug.")
```